src/components/dashboard/Relaying/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { WidgetBody, WidgetContainer } from '@/components/dashboard/styled'
+import { Box, Card, Divider, Grid, Link, Skeleton, SvgIcon, Typography } from '@mui/material'
+import RelayerIcon from '@/public/images/common/relayer.svg'
+import css from './styles.module.css'
+import useRemainingRelays from '@/hooks/useRemainingRelays'
+import useChains, { useCurrentChain } from '@/hooks/useChains'
+import { FEATURES, hasFeature } from '@/utils/chains'
+import ImageFallback from '@/components/common/ImageFallback'
+
+const FallbackChainIcon = ({ color }: { color: string }) => (
```

Comment:
The idea of using a circle with the chain color as fallback was mine. I will discuss it with Design anyway.

Reply:
I have presented it to Lila and being approved